tests/legacy/test_container_operations.py

```diff
@@ -81,6 +81,7 @@ def test_equality_operations(self, task, any_x_series_device, seed):
         assert ai_channel_1 != ai_channel_2
 
     @pytest.mark.parametrize("seed", [generate_random_seed()])
+    @pytest.mark.library_only
     def test_hash_operations(self, generate_task, any_x_series_device, seed):
```

Comment:
> The test_hash_operations test only can be tested with a task instance created through the c functions calls (libaray interpreter), since the hash value cannot be obtained for a grpc task instance. Therefore, this test has been marked to work with only library interpreter implementation with the use of library_only marker.

gRPC tasks should support the same operations as library tasks. Please file a bug and add a comment with a link/reference to the bug.

Reply:
I have added a comment to the test case related to the bug https://ni.visualstudio.com/DevCentral/_workitems/edit/2388532